recipes/litehtml/all/conanfile.py

```diff
@@ -43,6 +43,8 @@ def export_sources(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.version != "cci.20211028":
```

Comment:
```suggestion
        if Version(self.version).major == 0 and Version(self.version).minor >= 6:
```

https://github.com/litehtml/litehtml/pull/180/files